shared/css/common.css

```diff
@@ -591,3 +591,17 @@ html[dir="rtl"] .context-wrapper > .context-preview {
 .clicks-allowed.context-wrapper:hover > .context-info > .context-url {
   text-decoration: underline;
 }
+
+.remote-video-box {
+  height: 100%;
+  width: 100%;
+}
+
+.remote-video-box > .remote-cursor {
+  background: url('../img/cursor.svg#cursor-orange') no-repeat;
+  height: 20px;
+  position: absolute;
```

Comment:
This was tricky to track down. Turns out the svg cursor has a white outline (similar to what OS X has around the black cursor). The colored point should be what we actually point at, so we need a `margin: -2px` here to have things match up better. Also include a comment to why we need the negative margin.